SpatialGDK/Documentation/content/spatialos-concepts/schema.md

```diff
@@ -7,25 +7,45 @@ Schema is defined in `.schema` files and written in schemalang.  When you use th
 
 #### How to generate schema
 
-To generate schema, select the **Schema** button in the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through classes with replicated properties to generate the required schema files and then updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase).
+There are two ways to generate schema for your project:
 
-![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)
+* **Generate schema (Full Scan)** 
 
-_Image: In the GDK toolbar in the Unreal Editor, select **Schema**_
+    To generate schema for all objects with [replicated properties (Unreal documentation)](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties) in your project, in the Unreal Editor, on the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons), open the **Schema** drop-down menu and select **Schema (Full Scan)**.<br/> You must select **Schema (Full Scan)** the first time you generate schema for a project. 
```

Comment:
We also generate schema for objects that have RPCs. Here's unreal documentation on them - https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/RPCs

Reply:
It's very much a nit, as 99% of the time any class that has an RPC will have replicated properties. You also don't need to regenerate schema when modifying RPCs, we just use the existence of RPCs to ensure the core data schema component is generated in the first place.